.clitest/znap.source.md

```diff
@@ -0,0 +1,40 @@
+Initialize
+```zsh
+% source .clitest/__init__.zsh
+% .test.setup
+%
+```
+
+Znap! `source` command with no args prints usage:
+```zsh
+% znap source  #=> --exit 64
+znap source: not enough arguments
+Usage: znap source <repo> [ <dir> | <file> ] ...
+Load plugins.
+%
+```
+
+Sourcing a path loads its plugin file:
+```zsh
+% mkdir -p ~znap/foo
+% echo "echo sourcing foo..." > ~znap/foo/foo.plugin.zsh
```

Comment:
Why are you not using `.test.makeplugin` here?

Reply:
Good catch.